modules/caddyhttp/reverseproxy/selectionpolicies.go

```diff
@@ -699,8 +705,10 @@ func (s CookieHashSelection) Select(pool UpstreamPool, req *http.Request, w http
 //
 //	lb_policy cookie [<name> [<secret>]] {
 //		fallback <policy>
+//		[max_age <duration>]
```

Comment:
All subdirectives are implicitly optional, so the `[]` isn't needed here

```suggestion
//		max_age <duration>
```